File: YEAR2/SEM2/MPP/ConsoleApp1/domain/Participant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp1.domain
{
    public class Participant:Entity<long>
    {
        public string Nume { get; set; }

        public int Varsta { get; set; }

        public List<Proba> Proba { get; set; }


        public Participant(string Nume, int Varsta, List<Proba> Proba)
        {
            this.Nume = Nume;
            this.Varsta = Varsta;
            this.Proba = Proba;
        }
        public override string ToString()
        {
            return Nume+" "+Varsta;
        }
    }
}

File: YEAR2/SEM2/MPP/Tema1/src/main/java/org/example/domain/Participant.java
package org.example.domain;

import java.util.List;

public class Participant  extends Entity<Long> {

    private String nume;
    private int varsta;
    private List<Proba> probe;

    public Participant(String nume, int varsta, List<Proba> probe) {
        this.nume = nume;
        this.varsta = varsta;
        this.probe = probe;
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public int getVarsta() {
        return varsta;
    }

    public void setVarsta(int varsta) {
        this.varsta = varsta;
    }

    public List<Proba> getProbe() {
        return probe;
    }

    @Override
    public String toString() {
        return "Participant{" +
                "nume='" + nume + '\'' +
                ", varsta=" + varsta +
                '}';
    }

    public void setProbe(List<Proba> probe) {
        this.probe = probe;
    }
}
